bin/jpm

```diff
@@ -38,7 +38,8 @@ program
   .option("--debug", "Enable the add-on debugger when running the add-on")
   .option("--check-memory", "Enable leaked tracker that attempts to report compartments leaked")
   .option("--profile-memory", "Enable profiling of memory usage")
-  .option("--retro", "In development flag for transitioning to new style addons; forces the lack of install.rdf/bootstrap.js creation regardless of what engine versions are running");
+  .option("--retro", "In development flag for transitioning to new style addons; forces the lack of install.rdf/bootstrap.js creation regardless of what engine versions are running")
+  .option("--out <fname>", "Save xpi file with this name. Use with `jpm xpi`.");
```

Comment:
@kumar303 @matraska23 I would prefer an option name which is more specific than "--out", even better if it doesn't start with an 'o' (which is already used in the "-o, --overloading" option)

e.g. "--xpi-out-path"